File: ios/MyModule.m
//
//  MyModule.m
//  NativeView
//
//  Created by Dhayalu on 28/03/23.
//

#import <Foundation/Foundation.h>
#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(MyModule, NSObject)

//RCT_EXTERN_METHOD(zelleData: (NSString *)appName baseUrl:(NSString *)baseUrl instId:(NSString *)instId product:(NSString *)product ssoKey:(NSString *)ssoKey parameters:(NSDictionary *)parameters pd:(NSDictionary *)pd )

RCT_EXTERN_METHOD(zelleData: (NSString *)baseUrl instId:(NSString *)instId )


@end

File: ios/MyModule.swift
//
//  MyModule.swift
//  NativeView
//
//  Created by Dhayalu on 28/03/23.
//

import Foundation
import UIKit
import React.RCTBridgeModule


@objc(MyModule)
class MyModule: NSObject{

  var callback: RCTResponseSenderBlock?
  static var baseURL = ""

  //@objc func zelleData(_ appName: String, baseUrl: String, instId: String, product: String, ssoKey:String, parameters: NSDictionary, pd: NSDictionary){
  @objc func zelleData(_ baseUrl: String, instId: String){


    MyModule.baseURL = baseUrl

    NSLog("appname %@  baseUrl %@", baseUrl, instId)

  }


}
